src/components/teaser/teaser.stories.ts

```diff
@@ -66,70 +59,156 @@ const href: InputType = {
   },
 };
 
+const imgWidth: InputType = {
+  control: {
+    type: 'number',
+  },
+};
+
+const imgAspectRatio: InputType = {
+  control: {
+    type: 'number',
+  },
+};
```

Comment:
Not sure if we should provide controls as there are no props. maybe better to just have an example with different values but only in templates?

Reply:
There are no props but they are still used by the user to set width and aspect-ratio, so I'd prefer to keep them

Reply:
We can keep it, but would be great if we write somewhere that this in responsibility of the consumer and better don't use css vars in docs (see below), as they are not necessary.